src/components/credit.tsx

```diff
@@ -15,20 +15,24 @@ interface Props {
 	format: Format;
 }
 
-const styles = css`
+const mediaStyles = css`
 	${textSans.xsmall()}
 	margin: ${remSpace[1]} 0;
 `;
 
+const defaultStyles = css`
+	${textSans.xxsmall()}
+`;
+
 const Credit: FC<Props> = ({ format, credit }) =>
 	pipe(
 		credit,
 		map((cred) => {
 			switch (format.design) {
 				case Design.Media:
-					return <p css={styles}>{cred}</p>;
+					return <p css={mediaStyles}>{cred}</p>;
 				default:
-					return <> {cred}</>;
+					return <span css={defaultStyles}> {cred}</span>;
```

Comment:
What's the reason that one is a `span` and the other a `p`?

Reply:
The existing one was a `p` so i thought there must be a reason for it but I think a `span` makes more sense?